Source/WebCore/platform/animation/Animation.cpp

```diff
@@ -135,7 +146,9 @@ bool Animation::animationsMatch(const Animation& other, bool matchProperties) co
         && m_timingFunctionSet == other.m_timingFunctionSet
         && m_compositeOperationSet == other.m_compositeOperationSet
         && m_allowsDiscreteTransitionsSet == other.m_allowsDiscreteTransitionsSet
-        && m_isNone == other.m_isNone;
+        && m_isNone == other.m_isNone
```

Comment:
none should probably remain the last check (or moved to first).

Reply:
Fixed.